test/unit/modules/blocks/verify.js

```diff
@@ -455,14 +456,140 @@ describe('blocks/verify', () => {
 	describe('__private.verifyVersion', () => {
 		let verifyVersion;
 
-		it('should return error when block version > 0', () => {
-			verifyVersion = __private.verifyVersion({ version: 3 }, { errors: [] });
-			return expect(verifyVersion.errors[0]).to.equal('Invalid block version');
+		describe('when there are no exceptions for block versions', () => {
+			describe('when block height provided', () => {
```

Comment:
Does this mean that we can have block versions like 1, 3, 5 without including 2,4?

Reply:
Yes.